api-reference/environments/update-env-by-id.mdx

```diff
@@ -1,3 +1,67 @@
 ---
 openapi: put /v1/environments/{environmentId}
----
\ No newline at end of file
+---
+
+<RequestExample>
+```javascript Node.js
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<NOVU_API_KEY>');
+await novu.environmets.updateOne("environmentId", {
+  name: "Stagging" // optional
+  parentId: "parentEnvironmentId", // optional
+  identifier: "environmentIdentifier" // optional
+});
+```
+
+```php PHP
+use Novu\SDK\Novu;
+
+$novu = new Novu(<NOVU_API_KEY>);
+
+$novu->getCurrentEnvironment()->toArray();
+$novu->createEnvironment([
+    'name' => '<insert-name>',
+    'parentId' => '<insert-parent-id>' // optional
+])->toArray();
+```
+
+```ruby Ruby
+require 'novu'
+
+client = Novu::Client.new('<NOVU_API_KEY>')
+
+client.update_environment('64a713bdexxxxxx', {
+    'name' => 'Local', # optional
+    'identifier' => 'local', # optional
+    'parentId' => '7789', # optional
+    'dns' => { # optional
+        'inboundParseDomain' => 'dev.test' # optional
+    }
+})
+```
+
+```java Java
+import co.novu.sdk.Novu;
+
+public class Main {
+    public static void main(String[] args) {
+        String apiKey = "<NOVU_API_KEY>";
+
+        Novu novu = new Novu(apiKey);
+        Novu.updateEnvironment(environmentId, body);
```

Comment:
environmentId and body needs to be declared. However, we can ignore this for now, but we should note it down that we need to tag the Java maintainers to help with it once we are live.